index.html

```diff
@@ -0,0 +1,70 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+
+    <div>
+        <nav id="navbar">
+            <ul class="navlinks">
+                <li>Home</li>
+                <li>Projects</li>
+                <li>About</li>
+            </ul>
+        </nav>
+        <div id="welcome-section">
+            <h1>Abu bakar Siddique</h1>
+        </div>
```

Comment:
@abubakar83, nice use of inline comments here. However, when you are suggesting a change always include the reasons why you are suggesting it. 👍 I would advise you to edit your comment and include the reason why you are suggesting the change. 👍 

### Example of a good comment

- *Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible*

### Example of a bad comment
- *Why don't you use an article tag here instead of a div?*

Reply:
@Gambit142 I got your point and edit my commits.